admin/static/js/pages/prereg-admin-page.js

```diff
@@ -0,0 +1,254 @@
+*/
+var Assignee = function(reviewers) {
+    var self = this;
+    self.edit = ko.observable(false);
+
+    if (reviewers[0] !== 'none') {
+        reviewers.unshift('none'); 
+    }
+    
+    self.reviewers = ko.observableArray(reviewers);
+    self.assignee = ko.observable('none');
+};
+
+Assignee.prototype.editItem = function() {
+    var self = this;
+    self.assignee.edit(!self.editing);
+    self.editing(!self.editing);
+    
+};
+
+/**
+* Proof of publication column.
+*/
+var ProofOfPub = function() {
+    var self = this;
+    self.edit = ko.observable(false);
+    self.proofOfPub = ko.observable('Published Article Not Yet Submitted');
+    self.proofOfPubList = ko.observableArray(['Published Article Not Yet Submitted', 'Published Article Submitted', 'Published Article Under Review', 'Published Article Approved', 'Published Article Rejected']);
+};
+
+ProofOfPub.prototype.editItem = function() {
+    var self = this;
+    self.proofOfPub.edit(!self.editing);
+    self.editing(!self.editing);
+};
+
+/**
+* Payment sent column.
+*/
+var PaymentSent = function() {
+    var self = this;
+    self.edit = ko.observable(false);
+    self.paymentSent = ko.observable('no');
+    self.paymentSentList = ko.observableArray(['Yes', 'No']);
+};
+
+PaymentSent.prototype.editItem = function() {
+    var self = this;
+    self.paymentSent.edit(!self.editing);
+    self.editing(!self.editing);
+};
+
+/**
+* Notes column.
+*/
+var Notes = function() {
+    var self = this;
+    self.edit = ko.observable(false);
+    self.notes = ko.observable('none');
+};
+
+Notes.prototype.enlargeIcon = function(data, event) {
+    var icon = event.currentTarget;
+    $(icon).addClass("fa-2x");
+};
+
+Notes.prototype.shrinkIcon = function(data, event) {
+    var icon = event.currentTarget;
+    $(icon).removeClass("fa-2x");
+};
+Notes.prototype.editItem = function() {
+    var self = this;
+    self.editing(true);
+    self.notes.edit(true);
+};
+
+Notes.prototype.stopEditing = function() {
+    var self = this;
+    self.editing(false);
+    self.notes.edit(false);
+};
+
+/**
+* The row containing the information for a single draft displayed in columns.
+*
+* @param  {Dict}    params     The draft information
+* @param  {String}  permission Whether the admin can assign a reviewer
+* @param  {List}    reviewers  The prereg reviewers who can be assigned
+*/
+var Row = function(params, permission, reviewers) {
+    var self = this;
+
+    self.params = params;
+    self.viewingDraft = ko.observable(false);
+    if (permission === "false") {
+        permission = false;
+    } else {
+        permission = true;
+    }
+    self.adminPermission = ko.observable(permission);
+
+    self.editing = ko.observable(false);
+
+    self.title = params.registration_metadata.q1.value || 'no title';
+    self.fullname = params.initiator.fullname;
+    self.username = params.initiator.emails[0].address;
+    self.initiated = self.formatTime(params.initiated);
```

Comment:
It would be better to create `Date` instances and use the formatting method only in the template i.e. 
```self.initiated = new Date(params.initiated)``` and 
```<... data-bind="text: $root.formatDate(initiated)">```